api/config/v1alpha1/accesslogging_types.go

```diff
@@ -5,32 +5,41 @@
 
 package v1alpha1
 
-type ProxyAccessLogging struct {
-	// Format defines the format of access logging.
-	Format ProxyAccessLoggingFormat `json:"format"`
-	// Sinks defines the sinks of access logging.
+type ProxyAccessLog struct {
+	// Disable disables access logging for managed proxies if set to true.
+	Disable bool `json:"disable,omitempty"`
+	// Settings defines accesslog settings for managed proxies.
+	// If unspecified, will send default format to stdout.
+	// +optional
+	Settings []ProxyAccessLogSetting `json:"settings,omitempty"`
+}
+
+type ProxyAccessLogSetting struct {
+	// Format defines the format of accesslog.
+	Format ProxyAccessLogFormat `json:"format"`
+	// Sinks defines the sinks of accesslog.
 	// +kubebuilder:validation:MinItems=1
 	Sinks []ProxyAccessLoggingSink `json:"sinks"`
```

Comment:
should this be `ProxyAccessLogSink` as well ?

Reply:
yes, thanks for catching this